### PR TITLE
expat: add configure option --without-docbook to avoid docbook2X dependency

### PR DESCRIPTION
**Pull request by jose-d**
(created using `eb --new-pr`)

this is fix for bug #7927

### CODE REVIEW

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
koios1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/0eb53d88931f4525a4bb373002d9ad8c for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/d418aea8ff59b7a89f2e0ea8fff2362b for a full test report.

File: easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -18,6 +18,11 @@ source_urls = [SOURCEFORGE_SOURCE]
 sources = [SOURCELOWER_TAR_BZ2]
 checksums = ['17b43c2716d521369f82fc2dc70f359860e90fa440bea65b3b85f0b246ea81f2']
 
+# since expat 2.2.6, docbook2X is needed to produce manpage of xmlwf
+# docbook2X needs XML-Parser and XML-Parser needs expat
+# so circular dependency. "--without-docbook" breaks this circle.
+configopts = ['--without-docbook']
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb]: configopts go below builddeps, please fix asap, we need this for some other PR's deps

Comment (jose-d) [reply easybuild/easyconfigs/e/expat/expat-2.2.6-GCCcore-8.2.0.eb]: done.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/07a3d3508c0e63c54147414c526a26a8 for a full test report.

**Comment by akesandgren** [comment]
Odd travis failure, rekicking it.

**Comment by akesandgren** [comment]
Travis kicked

**Comment by akesandgren** [comment]
Kicking travis again, @jose-d was a bit premature on that :-)

**Comment by akesandgren** [comment]
Travis kicked

**Comment by akesandgren** [comment]
Going in, thanks @jose-d!

**Pull request merged**